onedal/linear_model/tests/test_linear_regression.py

```diff
@@ -71,15 +72,15 @@ def test_full_results(queue):
         model = LinearRegression(fit_intercept=True)
         model.fit(X, y, queue=queue)
 
-        assert_allclose(coef, model.coef_.T)
-        assert_allclose(intp, model.intercept_)
+        assert_allclose(coef, model.coef_.T, rtol=2e-3)
+        assert_allclose(intp, model.intercept_, rtol=1e-3)

```

Comment:
Is not rtol too big? What is justification of that?

Reply:
Changed